components/menu/index.css

```diff
@@ -182,35 +688,120 @@ governing permissions and limitations under the License.
   overflow: hidden;
 }
 
-.spectrum-Menu-checkmark {
-  display: none;
-  align-self: flex-start;
+.spectrum-Menu-sectionHeading {
+  color: var(--highcontrast-menu-section-header-color, var(--mod-menu-section-header-color, var(--spectrum-menu-section-header-color)));
+  display: block;
+  font-size: var(--mod-menu-section-header-font-size, var(--spectrum-menu-section-header-font-size));
+  font-weight: var(--mod-menu-section-header-font-weight, var(--spectrum-menu-section-header-font-weight));
+  line-height: var(--mod-menu-section-header-line-height, var(--spectrum-menu-section-header-line-height));
+  min-inline-size: var(--mod-menu-section-header-min-width, var(--spectrum-menu-section-header-min-width));
 }
 
-.spectrum-Menu-checkmark,
-.spectrum-Menu-chevron {
-  flex-grow: 0;
-  padding-inline-start: var(--spectrum-listitem-texticon-icon-gap);
-  margin-block-start: calc(var(--spectrum-listitem-texticon-ui-icon-margin-top) - var(--spectrum-listitem-texticon-padding-y) + 1px);
+/* collapsible sub-menus */
+.spectrum-Menu-item--collapsible {
+  &.is-open {
+    padding-block-end: 0;
+
+    .spectrum-Menu-chevron {
+      transform: rotate(90deg);
+    }
+  }
+
+  &.is-open:hover,
+  &.is-open:active,
+  &.is-open:focus,
+  &.is-open.is-focused {
+    background-color: var(--highcontrast-menu-item-background-color-default, var(--mod-menu-item-background-color-default, var(--spectrum-menu-item-background-color-default)));
```

Comment:
This background color changes in WHCM. What about using a `Background` or transparent for the background-color and `ButtonText` for the text color?

Reply:
Yes, thank you for this! Done.